src/course-unit/hooks.jsx

```diff
@@ -74,7 +79,13 @@ export const useCourseUnit = ({ courseId, blockId }) => {
     handlePreview: () => {
       window.open(draftPreviewLink, '_blank');
     },
-    handleEdit: () => {},
+    handleEdit: () => {
+      window.postMessage({
```

Comment:
[nit]: Can we send event via `sendMessageToIframe`?

Reply:
Replaced with sendMessageToIframe

Reply:
Resolved